storage/block/types.go

```diff
@@ -66,9 +66,9 @@ type FilteredBlocksMetadataIter interface {
 // FetchBlockResult captures the block start time, the readers for the underlying streams, the
 // corresponding checksum and any errors encountered.
 type FetchBlockResult struct {
-	Start   time.Time
-	Readers []xio.SegmentReader
-	Err     error
+	Start  time.Time
```

Comment:
Any reason to retain this field (Start) now that each Block has it?

Reply:
I was thinking the same thing, but figured checking the array has values and then getting Start from the first element is a bit clunkier than retaining this field? I can remove it if preferred